Google.Ads.GoogleAds/examples/ShoppingAds/AddPerformanceMaxRetailCampaign.cs

```diff
@@ -735,7 +737,12 @@ public string Next()
                 )
             );
 
-            return operations;
+            // The list of operations must be sorted so that all asset
+            // operations come before all the asset group asset operations,
+            // otherwise the API will reject the request.
+            var sorted = operations.OrderBy(x => x.AssetOperation is null).ToList();
```

Comment:
Could you just return the output of the RHS directly instead of reassigning to a variable then returning?